docs/pages/blog.tsx

```diff
@@ -0,0 +1,441 @@
+import HeroEnd from 'docs/src/components/home/HeroEnd';
+import Link from 'docs/src/modules/components/Link';
+
+export const getStaticProps = async () => {
+  const data = getAllBlogPosts();
+  return {
+    props: data,
+  };
+};
+
+const PostPreview = (props: BlogPost) => {
+  return (
+    <React.Fragment>
+      {props.tags && (
+        <Box sx={{ display: 'flex', gap: 1, mb: 1.5 }}>
+          {props.tags.map((tag) => (
+            <Chip
+              key={tag}
+              label={tag}
+              size="small"
+              sx={{
+                fontWeight: 500,
+                color: (theme) =>
+                  theme.palette.mode === 'dark' ? theme.palette.grey[50] : theme.palette.grey[700],
+                background: (theme) =>
+                  theme.palette.mode === 'dark'
+                    ? alpha(theme.palette.grey[700], 0.5)
+                    : theme.palette.grey[100],
+                '&:hover': {
+                  background: (theme) =>
+                    theme.palette.mode === 'dark'
+                      ? alpha(theme.palette.grey[700], 0.5)
+                      : theme.palette.grey[100],
+                },
+              }}
+            />
+          ))}
+        </Box>
+      )}
+      <Typography
+        component="h2"
+        fontWeight="bold"
+        variant="subtitle1"
+        sx={{
+          mb: 0.5,
+        }}
+      >
+        <Link
+          aria-describedby={`describe-${props.slug}`}
+          href={`/blog/${props.slug}`}
+          sx={{
+            color: 'text.primary',
+            '&:hover': {
+              textDecoration: 'underline',
+            },
+          }}
+        >
+          {props.title}
+        </Link>
+      </Typography>
+      <Typography color="text.secondary" sx={{ mb: 'auto' }}>
+        {props.description}
+      </Typography>
+      {props.authors && (
+        <AvatarGroup
+          sx={{
+            mt: 2,
+            mb: 1,
+            alignSelf: 'flex-start',
+            '& .MuiAvatar-circular': {
+              width: '28px',
+              height: '28px',
+              border: 3,
+              borderColor: (theme) =>
+                theme.palette.mode === 'dark'
+                  ? theme.palette.primaryDark[800]
+                  : theme.palette.grey[100],
+              backgroundColor: (theme) =>
+                theme.palette.mode === 'dark'
+                  ? theme.palette.primaryDark[700]
+                  : theme.palette.grey[100],
+              color: (theme) =>
+                theme.palette.mode === 'dark'
+                  ? theme.palette.primaryDark[100]
+                  : theme.palette.grey[800],
+              fontSize: (theme) => theme.typography.pxToRem(13),
+              fontWeight: 500,
+            },
+          }}
+        >
+          {(props.authors as Array<keyof typeof AUTHORS>).map((author) => (
+            <Avatar key={author} alt="" src={AUTHORS[author]?.avatar} />
```

Comment:
Use the author name as the alternative text?

Reply:
Done. Thanks for the heads up!